.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```

Comment:
One aspect of templates I've always been wary of is that it scares off issues that otherwise wouldn't be created. For example I've stopped filing a bug halfway through sometimes when the issue has seemingly a lot of fields that require a good deal of effort to fill in. For Cargo I'd personally prefer that we sift through low-quality reports than get no reports as there's always a chance that we can spot an actually issue from an ill-reported bug.

In that sense I feel that this sort of section (and maybe additional context?) goes into a bit too much information for a feature request. I'm curious what others think though?